prisma/schema.prisma

```diff
@@ -111,3 +111,24 @@ model EventRepairer {
 
   @@unique([userId, eventId])
 }
+
+enum Role {
+  REPAIRER
+  EVENT_MANAGER
+  ORG_ADMIN
+  SUPER_ADMIN
+}
```

Comment:
I think it may break things if we do wish to change the names.

Reply:
I guess I prefer the existing types? It's debatable what to call an org admin. Also, the user role should have Client incase the org admin wants to demote someone. Whatever you choose, I think it's enough pulling the enum from the prisma schema and not duplicating the types?

Reply:
From our discussion on saturday, I was under the impression that each `Staff` is just an elevated `User` (non-customer) and therefore has different roles? Actually, now that I think about it, probably doesn't matter tbh and could just use the existing `role`.

Reply:
In that case, then this Role refers to a staff role in the db and won't have a CLIENT, and UserRole would be the role fetched from clerk, which could? Maybe removing CLIENT from the UserRole type and treating no-roles as clients in the code will clean things up

Reply:
That sounds ideal

Reply:
soo what I need to do is remove client from userRole and then use userRole instead of Role? Or do I just make the names in Role match the names in userRole?